src/styles.css

```diff
@@ -337,6 +353,22 @@ form > ul > li > input {
   padding-left: 10px;
 }
 
+@media (max-width: 992px) {
+  .footer_divider {
+    display: none;
+  }
+}
+
+@media (min-width: 992px) {
+  .email {
+    display: none;
+  }
+
+  .footer_divider {
+    display: block;
+  }
+}
```

Comment:
- [ ] Kindly your cards in the desktop version should have the same size and styling as in the mobile version. Please work on the size(width and height)of the card in the desktop version they should look the same as template cards.

 Your cards 
-------|
![image](https://user-images.githubusercontent.com/65068771/149531421-b6a19be9-b3ae-4836-8c8a-21d905381ec8.png)
 the template cards 
![image](https://user-images.githubusercontent.com/65068771/149531664-4739b34c-ac91-4940-9318-e72bb0d718a0.png)